language/de/help/bbcode.php

```diff
@@ -52,7 +52,7 @@
 	'HELP_BBCODE_OTHERS_CUSTOM_ANSWER'	=> 'Wenn du ein Administrator mit entsprechenden Rechten auf diesem Board bist, kannst du im Bereich für benutzerdefinierte BBCode-Tags weitere Tags definieren.',
 	'HELP_BBCODE_OTHERS_CUSTOM_QUESTION'	=> 'Kann ich meine eigenen Tags hinzufügen',
 
-	'HELP_BBCODE_QUOTES_CODE_ANSWER'	=> 'Wenn du einen Auszug eines Programmcodes oder etwas anderes, das eine feste Textweite wie eine Courier-Schrift benötigt, eingeben möchtest, solltest du den Text in <strong>[code][/code]</strong> einschließen, z.&nbsp;B.<br /><br /><strong>[code]</strong>echo &quot;Das ist ein Stück Programmcode&quot;;<strong>[/code]</strong><br /><br />Alle Formatierungen, die innerhalb von <strong>[code][/code]</strong> genutzt werden, bleiben erhalten, wenn der Text betrachtet wird. Die Syntaxhervorhebung für PHP kann mit <strong>[code=php][/code]</strong> aktiviert werden; dies ist immer dann zu empfehlen, wenn PHP-Code gepostet wird, da so die Lesbarkeit verbessert wird.',
+	'HELP_BBCODE_QUOTES_CODE_ANSWER'	=> 'Wenn du einen Auszug eines Programmcodes oder etwas anderes, das eine feste Textweite wie eine Courier-Schrift benötigt, eingeben möchtest, solltest du den Text in <strong>[code][/code]</strong> einschließen, z.&nbsp;B.<br /><br /><strong>[code]</strong>echo &quot;Das ist ein Stück Programmcode&quot;;<strong>[/code]</strong><br /><br />Alle Formatierungen, die innerhalb von <strong>[code][/code]</strong> genutzt werden, bleiben erhalten, wenn der Text betrachtet wird.',
```

Comment:
Der Language-key wurde hier in dem PR nur gekürzt und sonst nicht verändert. Mach doch mal einen Vorschlag für eine kürzere Variante mit gleichem Inhaltsgehalt.